src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -188,6 +186,12 @@ private static Glyph GetGlyph(SymbolKind kind, Accessibility? declaredAccessibil
                 return (type, wasPlural);
             }
 
+            // The main purpose of this is to prevent converting "string" to "chars", but it also simplifies logic for other basic types (int, double, object etc.)
+            if (type.SpecialType != SpecialType.None)
+            {
+                return (type, wasPlural);
+            }
```

Comment:
how does it simplify the logic for the other tyeps?  it looks like those would just fall out from teh last line of thist method.  If we dont' want the string->chars issue, i would change *only* that.

Reply:
> it looks like those would just fall out from teh last line of thist method

It does, but after falling into the next `if` block and doing a lot of unnesesary actions there. Here is a demo for `float` without my change:
![devenv_jyClAXnBRg](https://user-images.githubusercontent.com/70431552/154992091-b9f3d330-7c3a-4c57-8d4d-0d81215a0185.gif)

So word "simplify" means mostly performance reasons here. However, I found a bug, that `IEnumerable<T>` is not recognized as speccial type:
![devenv_Cc2YnTChTm](https://user-images.githubusercontent.com/70431552/154992404-74b3ce26-e465-4b87-b9fe-27e5a100a14c.gif)
![devenv_dK1BwspHb6](https://user-images.githubusercontent.com/70431552/154992481-3ec73e98-b20c-45ba-ad63-892eb5d27c93.png)

I will tweak my logic to be correct even when the bug is fixed